include/eve/module/real/core/detail/generic/swizzle.hpp

```diff
@@ -35,7 +37,7 @@ namespace eve::detail
                                       ) noexcept
   {
     constexpr auto sz = Pattern::size();
-    return bit_cast( v.mask()[p], as<logical<wide<T,fixed<sz>>>>() );
+    return bit_cast( shuffle(v.mask(),p), as<logical<wide<T,fixed<sz>>>>() );
```

Comment:
Do we want to keep the name `swizzle` even if we use the `shuffle` function?

Reply:
swizzle is the bare bone of shuffle. I just moved it to where it belongs.
We can rename it to basic_shuffle